src/App.jsx

```diff
@@ -24,14 +25,19 @@ const App = () => {
 
   };
 
+  const removeTodo = (id) => {
+    const newTodoList = todoList.filter((todo) => todo.id === id)
+    setTodoList(newTodoList)
+  }
+
```

Comment:
I see that in running your code that the remove functionality isn't quite working correctly. Consider adding some more console.log statements to identify why the `todoList` isn't properly being filtered. 

I see the issue, and I'd be happy to answer any questions you have over Slack or in a 1:1 session. Feel free to ping me there if you want to chat more!